tests/functional/pointerevent_capture_suppressing_mouse-manual.js

```diff
@@ -0,0 +1,30 @@
+define(function(require) {
+	var registerSuite = require('intern!object');
+	var w3cTest = require('../support/w3cTest');
+	var name = 'pointerevent_capture_suppressing_mouse-manual';
+
+	registerSuite({
+		name: name,
+
+		main: function() {
+			return w3cTest(this.remote, name + '.html')
+				.findById('target0')
+					.moveMouseTo(50, 50)
+					.clickMouseButton(0)
+					.end()
+				.findById('target1')
+					.moveMouseTo(50, 30)
+					.clickMouseButton(0)
+					.end()
+				.findById('btnCapture')
+					.moveMouseTo(50, 4)
+					.pressMouseButton(0)
+					.end()
+				.findByTagName('body')
```

Comment:
Ok checked. These fall out naturally when moving the mouse from the `btnCapture` to the `body` coordinates. It's basically going down the page then back up. Without moving the left coordinate.